OpenCensusAgent/opencensus-service/receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -90,6 +97,55 @@ var errTraceExportProtocolViolation = errors.New("protocol violation: Export's f
 
 const receiverTagValue = "oc_trace"
 
+type mdsdJSON struct {
+	Tag    string   `json:"TAG"` // Tag is used to see if acknowledged by 1Agent
+	Source string   `json:"SOURCE"`
+	Data   []string `json:"DATA"` // Data must be an array
+}
+
+// SendTraceTo1Agent makes a connection to 1Agent and passes in the OpenCensus proto trace in JSON format
+func SendTraceTo1Agent(td *data.TraceData) {
+	log.Println("Starting funnel of data.")
+
+	conn, err := net.Dial("unix", "/var/run/mdsd/default_json.socket")
```

Comment:
make the socket path configurable. (either command line argument or env variable)

Reply:
This is a good idea. I will address this in another PR.